README.md

```diff
@@ -28,25 +32,32 @@ TODO: Write usage instructions here
 
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `bundle exec rake` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`.
 
 ### Run a development server
 
 ```sh
 $ bundle exec rake arclight:server
 ```
 
+Then visit http://localhost:3000.
+
 ### Run the test suite
 
+Ensure Jetty and Rails are not running, then:
```

Comment:
This now uses solr_wrapper instead of jetty. Perhaps we can switch out jetty for something else here?